tox.ini

```diff
@@ -3,11 +3,13 @@ envlist = ui-tests, flake8
 skipsdist = True
 
 [testenv:ui-tests]
-passenv = DISPLAY PYTEST_ADDOPTS
+passenv = DISPLAY PYTEST_ADDOPTS MOZ_HEADLESS SKIP_INSTALL_TEST
 basepython = python3.6
 deps = -rfrontend/test/ui/requirements/requirements.txt
+whitelist_externals =
+    bash
 commands =
-    bash bin/circleci/run-integration-tests.sh
+    bash bin/circleci/start-server.sh
```

Comment:
Still would kind of like to see this run from circle.yml instead of tox.ini, but it might not work. Might be fine to merge this as-is and tinker with that nit-pick later

Reply:
Me too, but when I tried it, I got a requests error, meaning the server was not found.

Reply:
Ah, yeah, probably circleci killing the old shell where it was in the background - whereas tox is all in the same shell.